docker-compose-production.yml

```diff
@@ -0,0 +1,58 @@
+
+version: '3'
+services:
+  postgres:
+    image: postgres:11
+    ports:
+      - "5432:${DB_PORT}"
+    environment:
+      POSTGRES_USER: ${DB_USERNAME}
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_DB: ${DB_DATABASE}
+    networks:
+      - infra_postgres
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+
+  server:
+    build:
+      dockerfile: "./Dockerfile"
+      context: "server/"
+    ports:
+      - "3000:${SERVER_PORT}"
+    environment:
+      SERVER_PORT: ${SERVER_PORT}
+      DB_TYPE: ${DB_TYPE}
+      DB_HOST: "postgres"
+      DB_PORT: ${DB_PORT}
+      DB_USERNAME: ${DB_USERNAME}
+      DB_PASSWORD: ${DB_PASSWORD}
+      DB_DATABASE: ${DB_DATABASE}
+    networks:
+      - infra_postgres
+      - infra_api
+    depends_on: ["postgres"]
+
+  client:
+    build:
+      dockerfile: "./Dockerfile"
+      context: "client/"
+    ports:
+      - "9000:${CLIENT_PORT}"
+      - "8080:8080"
```

Comment:
what is this port used for? not mentioned elsewhere in this PR and the nginx configuration only has a `listen 9000`

Reply:
Again, I didn't mean to commit this file, I will remove it